Kudu.Services/Diagnostics/ProcessController.cs

```diff
@@ -222,6 +236,199 @@ public HttpResponseMessage MiniDump(int id, int dumpType = 0, string format = nu
             }
         }
 
+        [HttpPost]
+        public async Task<HttpResponseMessage> StartProfileAsync(int id)
+        {
+            using (_tracer.Step("ProcessController.StartProfileAsync"))
+            {
+                // Check if the process Id exists in the sandbox. If it doesn't, this methid returns a 404 and we are done.
```

Comment:
Also, this comment looks like a copy paste error? I don't see where you're returning a 404